knowledge-base/kb-gridview-filteringdisctinctvalues.md

```diff
@@ -0,0 +1,48 @@
+---
+title: RadGridView Column Filtering Enum Values
+description: "How to filter Enum values in RadGridView column."
+type: how-to
+page_title: Filtering distinct values in RadGridView column for WPF.
+slug: kb-gridview-filtering-disctinctvalues
```

Comment:
The slug should contain the name of the control in lowercase and each word should be separated with a hyphen "-". 

In this case, we could change it to "kb-gridview-filtering-distinct-values"